services/recipeservice/src/main/java/ch/whatshouldieat/services/recipeservice/service/RecipeService.java

```diff
@@ -0,0 +1,130 @@
+package ch.whatshouldieat.services.recipeservice.service;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import ch.whatshouldieat.services.recipeservice.dao.RecipeRepository;
+import ch.whatshouldieat.services.recipeservice.model.Recipe;
+
+
+@Service @Transactional
+public class RecipeService {
+
+    private final RecipeRepository repository;
+    private List<Recipe> fullRecipesList;
+    
+   @Autowired
+   public RecipeService(RecipeRepository recipeRepository){
+       this.repository = recipeRepository;
+       initalizeDatabaseWithDemoValues();
```

Comment:
You write everytime the service class is instantiated these values to the database?

Reply:
Just for now to populate db with some demo values. Will add an TODO to change that in the future.

Reply:
added //TODOs .. This can be regarded as technical debt for now. Will resolve this at a later point - promised ;-) 
Added new issue: #3